lib/vfscore/README.md

```diff
@@ -0,0 +1,148 @@
+# VFSCore: Unikraft's Virtual File System API
+
+`vfscore` (*Virtual File System Core*) is an internal library of Unikraft that provides a generic interface of system calls related to filesystem management.
+There are multiple types of filesystems available in Unikraft (e.g: `ramfs`, `9pfs`).
+
+## Organization
```

Comment:
I would name this section something a bit more clear, `internal structure`, `file organization` , something along those lines.